lib/read-wasm-browser.js

```diff
@@ -18,6 +18,6 @@ module.exports = function readWasm() {
   );
 };
 
-module.exports.initialize = input => {
+module.exports.initialize = (input) => {
```

Comment:
I'm not able to understand why... but on mozilla-central we don't add parens on single arguments.

I saw https://github.com/prettier/prettier/pull/3324 but I don't see allowParens usages in mozilla-central...

Reply:
m-c is using [`prettier@1.19.1`](https://searchfox.org/mozilla-central/rev/0b1543e85d13c30a13c57e959ce9815a3f0fa1d3/package.json#28), and the default value for `allowParens` [changed](https://prettier.io/blog/2020/03/21/2.0.0.html#change-default-value-for-arrowparens-to-always-7430httpsgithubcomprettierprettierpull7430-by-kachkaevhttpsgithubcomkachkaev) in the 2.0 release.

@ochameau Should we add `arrowParens: avoid` to our config to match the previous behaviour?

Reply:
Oh nice catch! (I'm so happy to have you around so knowledgable about all this!!)

Yes, it would be nice. This would avoid having some useless changes when updating source-map code in -mc.